gui/screens/messages.py

```diff
@@ -115,10 +143,102 @@ def accept_community_invite(self, community: str) -> 'CommunityScreen':
+        super().__init__('mainWindow_CreateChatView')
+        self._confirm_button = Button('createChatView_confirmBtn')
+        self._cancel_button = Button('mainWindow_Cancel_StatusButton')
+        self._create_chat_contacts_list = List('createChatView_contactsList')
+
+    @property
+    @allure.step('Get contacts')
+    def contacts(self) -> typing.List[str]:
+        return self._create_chat_contacts_list.get_values('title')
+
+    @allure.step('Select contact in the list')
+    def select_contact(self, contact: str):
+        assert driver.waitFor(lambda: contact in self.contacts), f'Contact: {contact} not found in {self.contacts}'
+        self._create_chat_contacts_list.select(contact, 'title')
+
+    @allure.step('Create chat by adding contacts from contact list')
+    def create_chat(self, members):
+        for member in members[0:]:
+            time.sleep(0.2)
+            self.select_contact(member)
+        self._confirm_button.click()
+        return ChatMessagesView().wait_until_appears()
+
+
+class ChatMessagesView(QObject):
+
+    def __init__(self):
+        super().__init__('mainWindow_ChatMessagesView')
+        self._group_chat_message_item = TextLabel('chatLogView_Item')
+        self._group_name_label = TextLabel('statusChatInfoButton')
+        self._more_button = Button('moreOptionsButton_StatusFlatRoundButton')
+        self._edit_menu_item = QObject('edit_name_and_image_StatusMenuItem')
+        self._leave_group_item = QObject('leave_group_StatusMenuItem')
+        self._message_field = TextEdit('inputScrollView_Message_PlaceholderText')
+
+    @property
+    @allure.step('Get group name')
+    def group_name(self) -> str:
+        return self._group_name_label.text
+
+    @property
+    @allure.step('Get group welcome message')
+    def group_welcome_message(self) -> str:
+        for delegate in walk_children(self._group_chat_message_item.object):
+            if getattr(delegate, 'id', '') == 'msgDelegate':
+                for item in walk_children(delegate):
+                    if getattr(item, 'id', '') == 'descText':
+                        return str(item.text)
+
+    @allure.step('Click more options button')
+    def open_more_options(self):
+        self._more_button.click()
+
+    @allure.step('Choose edit group name option')
+    def open_edit_group_name_form(self):
+        time.sleep(2)
+        self.open_more_options()
+        time.sleep(2)
+        self._edit_menu_item.click()
+        return EditGroupNameAndImagePopup().wait_until_appears()
+
+    @allure.step('Choose leave group option')
+    def leave_group(self):
+        time.sleep(2)
+        self.open_more_options()
+        time.sleep(2)
+        self._leave_group_item.click()
+        return LeaveGroupPopup().wait_until_appears()
+
+    @allure.step('Send message to group chat')
+    def send_message_to_group_chat(self, message: str):
+        self._message_field.type_text(message)
+        driver.nativeType(str('<Return>'))
```

Comment:
try to use driver.type(self._message_field.object, '<Return>') 
it is better to send action to a direct object than native

Reply:
For some reason it doesn't work here, only with nativeType

Reply:
'<Return>' - it is a string, you don't need to convert it again to a str()

Reply:
driver.nativeType('<Return>')